[debug] [2024-08-22T16:55:08.281Z] ----------------------------------------------------------------------
[debug] [2024-08-22T16:55:08.290Z] Command:       /Users/moneibakang/.nvm/versions/node/v18.20.3/bin/node /Users/moneibakang/.nvm/versions/node/v18.20.3/bin/firebase deploy --only functions
[debug] [2024-08-22T16:55:08.290Z] CLI Version:   13.15.2
[debug] [2024-08-22T16:55:08.291Z] Platform:      darwin
[debug] [2024-08-22T16:55:08.292Z] Node Version:  v18.20.3
[debug] [2024-08-22T16:55:08.320Z] Time:          Thu Aug 22 2024 18:55:08 GMT+0200 (Central Africa Time)
[debug] [2024-08-22T16:55:08.320Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-08-22T16:55:08.454Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-08-22T16:55:08.455Z] > authorizing via signed-in user (bakangmonei2@gmail.com)
[debug] [2024-08-22T16:55:08.455Z] [iam] checking project receipts-for-tuition for permissions ["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","firebase.projects.get"]
[debug] [2024-08-22T16:55:08.460Z] > refreshing access token with scopes: []
[debug] [2024-08-22T16:55:08.465Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-22T16:55:08.465Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-22T16:55:18.941Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------044836110221353118233382","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-22T16:55:18.942Z] [iam] error while checking permissions, command may fail: FirebaseError: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2024-08-22T16:55:18.943Z] > refreshing access token with scopes: []
[debug] [2024-08-22T16:55:18.944Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-22T16:55:18.944Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-22T16:55:29.287Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------094611113773108459730117","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-22T16:55:29.288Z] [functions] service account IAM check errored, deploy may fail: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci {"name":"FirebaseError","children":[],"exit":1,"message":"Authentication Error: Your credentials are no longer valid. Please run \u001b[1mfirebase login --reauth\u001b[22m\n\nFor CI servers and headless environments, generate a new token with \u001b[1mfirebase login:ci\u001b[22m","status":500}
[info] 
[info] === Deploying to 'receipts-for-tuition'...
[info] 
[info] i  deploying functions 
[debug] [2024-08-22T16:55:29.298Z] > refreshing access token with scopes: []
[debug] [2024-08-22T16:55:29.299Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-22T16:55:29.299Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-22T16:55:39.781Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------541714273507004965265241","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-22T16:55:39.782Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2024-08-22T16:55:44.804Z] Error: Timed out.
    at Timeout._onTimeout (/Users/moneibakang/.nvm/versions/node/v18.20.3/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
[error] 
[error] Error: An unexpected error has occurred.
